plugin/trino-example-plugin/pom.xml

```diff
@@ -0,0 +1,274 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.example</groupId>
```

Comment:
Sorry, I'm still confused why we need a different group ID? Doesn't the user normally specify the group ID when generating the archtype, so we can use anything and it will be replaced later?

Reply:
You're right, we don't need a different group ID. I was confused that the package name would get mixed up with the `io.trino.` prefix in imports, but I used a package name that's longer. I removed all other workarounds and everything looks much better now.